plugins/org.eclipse.xtend.core/src/org/eclipse/xtend/core/validation/XtendJavaValidator.java

```diff
@@ -932,6 +932,7 @@ private boolean contributesToConflict(JvmDeclaredType type, Set<JvmDeclaredType>
 	protected void reportMissingImplementations(XtendTypeDeclaration xtendClass, JvmGenericType inferredType, List<IResolvedOperation> operationsMissingImplementation) {
 		StringBuilder errorMsg = new StringBuilder();
 		String name = xtendClass.getName();
+		boolean needsNewLine = operationsMissingImplementation.size() > 1;
 		if (xtendClass.isAnonymous()) {
 			JvmTypeReference superType = Iterables.getLast(inferredType.getSuperTypes());
 			errorMsg.append("The anonymous subclass of ").append(superType.getSimpleName());
```

Comment:
You can add a commit to the same branch and hit "pull request" again. That
will open a PR with just the new change. "Amending" a PR after merge is not
possible AFAIK.

2015-03-19 10:46 GMT+01:00 Vlad Dumitrescu <notifications@github.com>:

> In
> plugins/org.eclipse.xtend.core/src/org/eclipse/xtend/core/validation/XtendJavaValidator.java
> <https://github.com/eclipse/xtext/pull/33#discussion_r26740167>:
>
> > @@ -932,6 +932,7 @@ private boolean contributesToConflict(JvmDeclaredType type, Set<JvmDeclaredType>
> >  	protected void reportMissingImplementations(XtendTypeDeclaration xtendClass, JvmGenericType inferredType, List<IResolvedOperation> operationsMissingImplementation) {
> >  		StringBuilder errorMsg = new StringBuilder();
> >  		String name = xtendClass.getName();
> > +		boolean needsNewLine = operationsMissingImplementation.size() > 1;
> >  		if (xtendClass.isAnonymous()) {
> >  			JvmTypeReference superType = Iterables.getLast(inferredType.getSuperTypes());
> >  			errorMsg.append("The anonymous subclass of ").append(superType.getSimpleName());
>
> Right! I fixed this now, but I'm not sure how to best submit it.
>
> If I add a commit to the original branch, it should be possible to merge
> again, but I don't know if GitHub will detect that the PR changed status.
> Alternatively, I can open a new PR.
>
> What do you think? Shall we try updating this one and see if it works?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eclipse/xtext/pull/33/files#r26740167>.
>



-- 
Stefan Oehme
Consultant

Telefon: +49 (0) 431 990 268  70
Mobil:   +49 (0) 151 561 676  77

oehme@itemis.de
https://www.xing.com/profile/Stefan_Oehme4

http://itemis.de
http://twitter.com/itemis


itemis AG
Niederlassung Kiel
Am Germaniahafen 1
24143 Kiel

Rechtlicher Hinweis:

Amtsgericht Dortmund, HRB 20621

Vorstand: Jens Wagener (Vors.), Wolfgang Neuhaus, Dr. Georg Pietrek,
Jens Trompeter, Sebastian Neus

Aufsichtsrat: Prof. Dr. Burkhard Igel (Vors.), Michael Neuhaus, Jennifer
Fiorentino